index.html

```diff
@@ -1048,22 +1055,87 @@
+
+Accounts with a service need to be protected more strongly than just with a username and
+password. Since a compromised account reveals all the private information stored in the
+account, this is a privacy issue and not just a security issue. Services often store the
+historical locations and machine characteristics that have accessed an account, in order
+to make it harder to log in from an unusual place or type of machine. They might also
+store other personal data, like a phone number or address, in order to check that a
+suspicious login actually comes from the real account owner.
 
-<aside class="issue">
-  This section has not been edited for narrative.
 </aside>
 
+It is attractive to say that if someone violates the norms of a service they're using,
+then they sacrifice a proportionate amount of their privacy protections, but
+
+1. Often the service can only prevent the norm violation by also collecting data from
+   innocent users. This extra collection is not always [=appropriate=], especially if it
+   allows pervasive monitoring ([[RFC7258]], [[RFC7687]]).
+1. If a service owner wants to collect some extra data, it can be tempting for them to
+   define norms and proportionality that allow them to do so.
+
+<div class="practice">
+
+<p><span class="practicelab" id="principle-limited-collection-for-safety">If a service
+collects extra data from its users in order to protect those or other users, it must take
+extra technical and legal measures to ensure that this data can't be then used for other
+purposes, like to grow the service.</span></p>
+
+This is a special case of the more general principle that data can't be used for more
+[=purposes=] than the data's subjects understood it was being collected for.
+
+</div>
 
-<aside class="issue" data-number="73">
+<aside class="example" id="example-technical-legal-measures" title="Technical and Legal Measures">
 
-Discuss how systems intended to protect a person might justifiably collect more data from
-that person and from people who might be trying to attack that person. Also discuss
-whether systems intended to protect server-side parties from abuse can collect more data
-from users.
+A site might segregate the data it collects for safety reasons from its business data by:
 
-Discuss what limits need to be placed on such practices to distinguish them from privatised
-law enforcement and intellligence agencies.
+1. Writing in its privacy policy that these types of data are kept separate. (A legal
+   measure)
+1. Using multi-party computation to ensure that its business side can't learn the
+   sensitive safety data unless both its safety side and a trusted third party collude. (A technical measure)
+1. Hiring a trusted auditor to publicly check that the algorithms are sufficient to
+   provide the guarantees they claim to provide, and that there isn't, in fact, any
+   collusion. (Another technical measure)
```

Comment:
Hiring an auditor seems like a policy measure, not "Another technical measure".

Reply:
Yeah, I wasn't sure how to describe this. It's systematic and operates automatically once it's set in motion, which resembles other technical measures. But it only matters if humans take action based on its outputs, for example by filing lawsuits or exerting social or market pressure. audits are to encryption as [ceremonies](https://eprint.iacr.org/2007/399.pdf) are to protocols? I'm happy to change the wording.